internal/biz/resources/resources.go

```diff
@@ -3,9 +3,10 @@ package resources
 import (
 	"context"
 	"errors"
-	"github.com/google/uuid"
 	"time"
 
+	"github.com/google/uuid"
```

Comment:
Seems like an unwanted change?

Reply:
yeah im going to fix it this morning, its there difference between code editors i think and how VS Code go fmt wants to shuffle it down and separate the github packages from the standard library. Ill edit in VIM so it doesnt do that